File: halcyon-oauth2/src/main/java/com/marshal/halcyon/oauth2/service/OauthAccessTokenService.java
package com.marshal.halcyon.oauth2.service;

import com.marshal.halcyon.core.service.BaseService;
import com.marshal.halcyon.oauth2.entity.OauthAccessToken;

import java.util.List;

public interface OauthAccessTokenService extends BaseService<OauthAccessToken> {

    OauthAccessToken addTokenLogs(OauthAccessToken token);

    List<OauthAccessToken> selectOauthAccessTokens(OauthAccessToken condition, int pageNum, int pageSize);
}

File: halcyon-web/src/main/resources/static/js/service/oauth2/oauthAccessTokenService.js
/**
 * author:Marshal
 * description:service层
 */
app.service("oauthAccessTokenService", function ($http) {
    //base crud
    this.query = function (pageNum, pageSize, condition) {
        return $http.post("../oauth2/accessToken/query?pageNum=" + pageNum + "&pageSize=" + pageSize, condition);
    };

    this.save = function (oauthAccessToken) {
        return $http.post("../oauth2/accessToken/submit", oauthAccessToken);
    };

    this.delete = function (selectedIds) {
        return $http.get("../oauth2/accessToken/remove?selectedIds=" + selectedIds);
    };

    this.queryById = function (id) {
        return $http.get("../oauth2/accessToken/queryById?id=" + id);
    }

});